cypress/plugins/db_teardown.ts

```diff
@@ -4,14 +4,16 @@ export default async () => {
   let prisma = new PrismaClient();
 
   try {
-    // delete all records, except users and sessions
+    // delete all records
     // HACK: posts must be deleted first,
     // in order to properly get rid of relations in the the DB
```

Comment:
Now that I think of it, we should cascade when a user is deleted to all the posts.